tutorial/branches/simultaneous-changes.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Handling simultaneous changes and conflicts
+permalink: /tutorial/branches/simultaneous-changes/
+---
+
+In the [previous section](/tutorial/branches/project-diff/) you examined the project diff to see what has been changed in the branch. You now know what changes will be applied when the branch is merged. But before you proceed to merging the development branch to production, there is one more scenario that needs to be mentioned. While you were working in your development branch, someone else might have made changes to your production branch in the meantime.
+
+*Note: If you skipped the [file manipulation part](/tutorial/branches/prepare-files) of the tutorial, you'll need to change your Snowflake transformation instead of the Python transformation.*
+
+Let’s see what this might look like. Go to your production branch, to your transformation, and edit the variable
+`outFileContent`. Instead of `Hello world!` write `Hello tester!`, and then click on the blue check icon.
+
+{: .image-popup}
+![Screenshot - Edit Variable in Production](/tutorial/branches/figures/22-edit-var-in-prod.png)
+
+Now we’ve made configuration changes in both branches. Switch back to the development branch and look at the
+project diff. You’ll see a **warning message** saying that there is a conflict between the two configurations.
+
+{: .image-popup}
+![Screenshot - Conflict Warning](/tutorial/branches/figures/23-conflict-warning.png)
+
+Click **Compare with production** to see the differences.
+
+{: .image-popup}
+![Screenshot - Configuration Changes Differences](/tutorial/branches/figures/24-config-changes-diff.png)
+
+The conflict here means that the production project changed while we were working in the development branch.
+`Hello world!` was changed to `Hello tester!`. If you merged the development project to production now,
+the development project would overwrite any changes made in production. In our case, `Hello tester!` would become
+`Hello world!` again.
+
+## Resolve Conflicts
+To resolve the conflict and merge the development branch to production safely without overwriting anything new, you have two options. First, you can just change the branch configuration to have an expected diff against the production. Or you can reset the branch configuration to production version and apply you changes again - this is the better and safer option. 
```

Comment:
když je to better and safer, tak je proč je to až druhá možnost?

Reply:
Protože když si to vyzkoušíš v opačném pořadí, tak ti to zmizí z conflicts a už si ten druhý scénář nevyzkoušíš :)

Reply:
no ja to vím, ale při čtení to smysl nedává :)

Reply:
anebo tady můžeš použít tu druhou branch